app/src/components/NextButton.tsx

```diff
@@ -0,0 +1,7 @@
+import MainButton from './MainButton'
+
+const NextButton = ({ nextPomodoro }: { nextPomodoro: () => void }) => {
```

Comment:
What is `nextPomodoro` prop?

Shouldn't it be just a standard onClick or something? :thinking:

Reply:
Could you move the prop to its interface?